CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Change Log
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
+## Version 0.7.0 (2019-09-16)
+### Added
+- 0.7.0 contains the first release of a new language service that we are creating specifically for Azure Resource Manager deployment template files.  Up to this point, the extension has been built on top of the built-in VS Code JSON language server.  This has caused some problems:
+  a) Deployment templates allow comments, unlike standard JSON files
+  b) Deployment templates allow multi-line strings
+  c) Deployment templates use case-insensitive properties
+  d) Deployment templates have looser type rules, allowing substitutions such as "true" and "false" instead of true and false
+  e) The large schema files published for Azure resources cause poor validation performance
+  f) The errors provided by standard JSON validation frequently provide poor suggestions for fixing (due to lack of knowledge of Azure Resource Manager-specific properties such as resource name and apiVersion)
```

Comment:
I like the idea of including d-f because it is essentially a roadmap. Any concerns with adding some of our other roadmap items like:
* snippets
* better support for UDFs and copy loops